### PR TITLE
PR16963 Follow-up

### PR DESCRIPTION
**Pull request by bmcelvee**
Follow-up to https://github.com/openshift/openshift-docs/pull/16963. 

Relates to https://issues.redhat.com/browse/OSDOCS-789.

### CODE REVIEW

**Comment by bmcelvee** [comment]
@ironcladlou PTAL - thanks!

**Comment by bmcelvee** [comment]
@lihongan Please review when you get a chance, thanks! 

Just to note, this is a follow-up to #16963.

**Comment by ironcladlou** [comment]
Nice, thanks!

/lgtm

**Comment by bmcelvee** [comment]
Thanks, Dan! Once the rest of the reviews are complete I will get this one merged.

**Comment by bmcelvee** [comment]
@openshift/team-documentation please peer review, thanks!

File: modules/nw-ingress-default-internal.adoc [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+// * networking/ingress-operator.adoc
+
+[id="nw-ingress-default-internal_{context}"]
+= Configuring the default Ingress Controller for your cluster to be internal
+
+You can configure the `default` Ingress Controller for your cluster to be internal by deleting and recreating it.
+
+[WARNING]
+====
+If your cloud provider is Azure, you must have at least one public load balancer that points to your nodes.
```
Comment (bobfuru) [inline modules/nw-ingress-default-internal.adoc]: s/Azure/Microsoft Azure/ (at least on first instance, I think)

File: modules/nw-ingress-default-internal.adoc [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+If you do not, all of your nodes will lose egress connectivity to the Internet.
```
Comment (bobfuru) [inline modules/nw-ingress-default-internal.adoc]: Alex and I discussed our lack of a guideline for "internet" vs. "Internet". Our docs use both.  ISG has weird guidance about using it both ways, but it's pretty outdated. Elsewhere, I most often see it lowercase, which I'm a fan of. (Check out goog's explanation: https://developers.google.com/style/word-list#letter-i)

File: modules/nw-ingress-default-internal.adoc [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+====
+
+[IMPORTANT]
+====
+If you want to change the `scope` for an `IngressController` object, you must delete and then recreate that `IngressController` object. You cannot change the `.spec.endpointPublishingStrategy.loadBalancer.scope` parameter after the Custom Resource (CR) is created.
+====
+
+.Prerequisites
+
+* Install the OpenShift Command-line Interface (CLI), commonly known as `oc`.
```
Comment (bobfuru) [inline modules/nw-ingress-default-internal.adoc]: s/Command-line Interface/command-line interface/
See: https://docs.openshift.com/container-platform/4.2/cli_reference/openshift_cli/getting-started-cli.html#cli-about-cli_cli-developer-commands

**Review by bobfuru — COMMENTED** [review]
A few minor things but otherwise LGTM!

File: modules/nw-ingress-default-internal.adoc (hunk 2 above)
Comment (bmcelvee) [reply modules/nw-ingress-default-internal.adoc]: Oh thank goodness. That's my preference too. :)

**Comment by lihongan** [comment]
LGTM, thanks

**Pull request merged**